### PR TITLE
Added upgrade docs

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/upgrade-stackrox/from-44/

Preview:
- https://openshift-docs-git-upgrade-docs-gnelson.vercel.app/openshift-acs/master/upgrading/upgrading_roxctl/upgrade-from-44.html
- https://openshift-docs-git-upgrade-docs-gnelson.vercel.app/openshift-acs/master/upgrading/upgrade-helm.html

Uses following modules from https://github.com/openshift/openshift-docs/pull/33302
- [Changing configuration options after deploying the central-services Helm chart](https://github.com/openshift/openshift-docs/blob/a8ab2de7f40fdc5e3f3d084e7bf2161178a1a228/modules/change-config-options-after-deployment-central-service.adoc)
- [Changing configuration options after deploying the secured-cluster-services Helm chart](https://github.com/openshift/openshift-docs/blob/a8ab2de7f40fdc5e3f3d084e7bf2161178a1a228/modules/change-config-options-after-deployment.adoc)

### CODE REVIEW

File: upgrading/upgrading_roxctl/upgrade-from-44.adoc [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+include::modules/upgrade-scanner.adoc[leveloffset=+2]
+
+include::modules/verify-central-cluster-upgrade.adoc[leveloffset=+2]
+
+[id="upgrade-secured-clusters"]
+== Upgrading all secured clusters
+
+After upgrading Central services, you must upgrade all secured clusters, including the Central cluster.
+//TODO: Confirm with Jamie ↑
```
Comment (gaurav-nelson) [inline upgrading/upgrading_roxctl/upgrade-from-44.adoc:43]: @IAmATeaPot418 Can you confirm this?

I think after we updated Central, we only need to update secure  clusters.

File: upgrading/upgrading_roxctl/upgrade-from-44.adoc [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+[IMPORTANT]
+====
+If you are using automatic upgrades:
+
+* Update all your secured clusters by using automatic upgrades
```
Comment (bergerhoffer) [inline upgrading/upgrading_roxctl/upgrade-from-44.adoc:49]: Missing a period

**Comment by bergerhoffer** [comment]
This also has a conflict now.

File: _topic_map.yml [hunk 3]
```diff
@@ -65,9 +65,21 @@ Topics:
...
+Name: Upgrading
+Dir: upgrading
+Distros: openshift-acs
+Topics:
+- Name: Upgrading using Helm charts
+  File: upgrade-helm
+- Name: Upgrading using roxctl CLI
+  Dir: upgrading_roxctl
+  Topics:
+  - Name: Upgrading from version 3.0.44 or higher
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply _topic_map.yml:79]: Yes, more assemblies for upgrading from different versions.

File: modules/rollback-central-forced.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* Before you can perform a rollback, you must have free disk space available on your persistent storage. {product-title} uses disk space to keep a copy of databases during the upgrade. If the disk space is not enough to store a copy and the upgrade fails, you will not be able to roll back to an earlier version.
+
+.Procedure
+
+* Run the following commands to perform a forced rollback:
+** To forcefully rollback to the previously installed version:
++
+[source,terminal]
+----
+$ oc -n stackrox edit configmap/central-config
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/rollback-central-forced.adoc:28]: Yeah, it's not. I fixed the command.

File: upgrading/upgrade-helm.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+* Update the Helm chart.
+* Update configuraition files for the central-services Helm chart.
+* Upgrade the central-services Helm chart.
+* Update configuraition files for the secured-cluster-services Helm chart.
+* Upgrade the secured-cluster-services Helm chart.
+
+include::modules/updating-helm-repository.adoc[leveloffset=+1]
+
+include::modules/change-config-options-after-deployment-central-service.adoc[leveloffset=+1]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply upgrading/upgrade-helm.adoc:19]: Yup, they were part of another one, it's merged now.

File: modules/updating-helm-repository.adoc [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+// * upgrading/upgrade-helm.adoc
+:_module-type: PROCEDURE
+[id="updating-helm-repository_{context}"]
+= Updating the Helm chart repository
+
+You must always update Helm charts before upgrading to a new version of {product-title}.
+
+.Prerequisites
+
+* You must have already xref:../installing/installing_helm/install-helm-customization.adoc#adding-helm-repository_acs-install-helm-customization[added the {product-title} Helm chart repository].
```
Comment (bergerhoffer) [inline modules/updating-helm-repository.adoc:12]: Have to remove this xref

File: upgrading/upgrading_roxctl/upgrade-from-44.adoc [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+[id="upgrade-secured-clusters"]
+== Upgrading all secured clusters
+
+After upgrading Central services, you must upgrade all secured clusters.
+
+[IMPORTANT]
+====
+* If you are using automatic upgrades:
+** Update all your secured clusters by using automatic upgrades.
+** Skip the instructions in this section and follow the instructions in the xref:#verify-last-check-in-time_{context}[Verifying last check-in time for secured clusters] and xref:revoke-the-api-token_{context}[Revoking the API token] sections.
```
Comment (bergerhoffer) [inline upgrading/upgrading_roxctl/upgrade-from-44.adoc:48]: For these xrefs, you'll need to do the full path (xref:../../\<path>/<file_name.adoc>#\<anchor>).

File: modules/update-other-images.adoc [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+:_module-type: PROCEDURE
+[id="update-other-images_{context}"]
+= Updating other images
+
+[role="_abstract"]
+If you are upgrading {product-title} version number between 3.0.44 and 3.0.54, you must update the Sensor and Collector images.
+
+[WARNING]
+====
+Updating images is only required when you are upgrading from {product-title} version number between 3.0.44 and 3.0.54. Otherwise, skip the instructions in this section.
```
Comment (bergerhoffer) [inline modules/update-other-images.adoc:13]: There are still a handful of this wording with "number" that could be changed to remove "number" and add "a"

File: upgrading/upgrading_roxctl/upgrade-from-44.adoc [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+include::modules/update-openshift-security-context-constraints.adoc[leveloffset=+2]
+
+include::modules/update-other-images.adoc[leveloffset=+2]
+
+include::modules/verify-secured-cluster-upgrade.adoc[leveloffset=+2]
+
+[id="rollback-central"]
+== Rolling back Central
+
+You can rollback to a previous version of Central if the upgrade to a new version is unsuccessful.
```
Comment (bergerhoffer) [inline upgrading/upgrading_roxctl/upgrade-from-44.adoc:65]: rollback -> roll back

**Review by bergerhoffer — COMMENTED** [review]
Just a few more things here

**Comment by bergerhoffer** [comment]
LGTM

**Pull request merged**